packages/core/src/UserState.ts

```diff
@@ -343,6 +343,7 @@ export default class UserState {
         const attestations = await this.sync._db.findMany('Attestation', {
             where: {
                 epoch,
+                attesterId: this.sync.attesterId.toString(),
```

Comment:
I prefer that this can be a input in 
```ts
public getEpochKeyIndex = async (
     epoch: number,
     _epochKey: bigint | string
 )
```
so that it can support multi-attesters synchronizer/user state

Reply:
I think this should be done either in the multi-attester pr, or by basing this branch on the multi-attester pr. Those changes aren't in main yet